i18n/troubleshooting_es.md

```diff
@@ -0,0 +1,28 @@
+---
+# Ups, algo no está funcionando.
+# ¡No desesperes!  
```

Comment:
I liked # Pero !Que no cunda el pánico!
it was kinda like an easter egg for el Chapulín Colorado watchers.